Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -502,10 +506,19 @@ open class NavigationMapView: UIView {
         
         if let lineLayer = lineLayer {
             do {
-                try mapView.style.addLayer(lineLayer,
-                                           layerPosition: isMainRoute ?
-                                            LayerPosition(above: mapView.mainRouteLineParentLayerIdentifier) :
-                                            LayerPosition(below: parentLayerIndentifier))
+                var layerPosition: MapboxMaps.LayerPosition? = nil
+                
+                if isMainRoute {
+                    if let aboveLayerIdentifier = mapView.mainRouteLineParentLayerIdentifier {
```

Comment:
If the identifier is nil, this method silently returns without adding a layer or printing to the console. Is that intentional? It’d be easier to keep track of cases like this by making `layerPosition` constant (but still optional).

Reply:
I've updated the code to use `let` instead of `var`. Otherwise code should work fine: if there is no `layerPosition` layer will be added at the top of the stack.